docs/semgrep-secrets/getting-started.md

```diff
@@ -1,153 +1,139 @@
 ---
 slug: /semgrep-secrets/getting-started
 append_help_link: true
-title: Scanning for secrets
+title: Scan for secrets
 hide_title: true
 description: Set up secrets scanning to find and rotate valid leaked secrets.
 tags:
   - Semgrep Secrets
-toc_max_heading_level: 2
 ---
 
 import MoreHelp from "/src/components/MoreHelp"
 
-# Getting started with Semgrep Secrets
+# Scan for secrets
 
-Detect and triage leaked secrets and credentials by scanning with Semgrep Secrets.
+Semgrep Secrets allows you to detect and triage leaked secrets and credentials
+and save time by prioritizing which secrets to rotate based on whether they're active and in use.
 
-Save time by prioritizing which secrets to rotate based on whether they are valid (active and in-use) or not.
+![Semgrep Secrets page](/img/secrets-page.png#md-width)
 
-![Semgrep Secrets page](/img/secrets-page.png)
-**_Figure._** Semgrep Secrets page. <br />
+This document guides you through:
 
-This document guides you through the following:
+1. Enabling Semgrep Secrets
+2. Viewing your results and triaging your findings
+3. Setting up PR comments and notifications
 
-* Enabling Semgrep Secrets
-* Receiving findings in GitHub PR comments
-* Triaging findings
+## Language and environment support
 
-To learn about how Semgrep Secrets detects secrets, see [<i class="fa-regular fa-file-lines"></i> Conceptual overview of Semgrep Secrets](/docs/semgrep-secrets/conceptual-overview).
+Semgrep Secrets can scan repositories using **any programming language** and supports the use GitHub, GitLab, and Bitbucket for PR and MR comments.
 
-:::info Feature maturity
-* This feature is in public beta.
-* You or your developers may encounter bugs. For issues, reach out to [<i class="fa-regular fa-envelope"></i> support@semgrep.com](mailto:support@semgrep.com).
-:::
+## Prerequisites
 
-:::tip Sign-up
-To gain access to this feature, fill out the [<i class="fas fa-external-link fa-xs"></i> Semgrep Secrets Beta](https://get.semgrep.dev/secrets-beta-request.html) form.
-:::
+Before proceeding, ensure that you have:
 
-## Supported developer environments and features
+* An [up-to-date installation of the Semgrep CLI](/getting-started/quickstart/). Semgrep Secrets does not work on older versions of Semgrep.
+* A Semgrep Cloud Platform account with an organization already set up
+* [Added or onboarded at least one
+repository to Semgrep Cloud Platform for
+scanning](/semgrep-cloud-platform/getting-started/#starting-a-sast-and-sca-scan-on-a-remote-repository).
 
-This section lists the source code managers (SCMs) that Semgrep Secrets supports.
+## Enable Semgrep Secrets
```

Comment:
Out of scope, but I could not help but notice we don't have instructions to enable or disable Semgrep Supply Chain or Semgrep Code! :exploding_head:  - we should probably add that in.

Reply:
https://linear.app/semgrep/issue/MKT-870/add-instructions-to-enable-or-disable-semgrep-supply-chain-or-semgrep